packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue

```diff
@@ -67,9 +67,21 @@ LibraryPanelKind: {
 }
 
 LibraryPanelSpec: {
+  // Panel ID for the library panel in the dashboard
+  id: number
+  // Title for the library panel in the dashboard
+  title: string
+  
+  libraryPanel: LibraryPanelRef
+}
+
+// A library panel is a reusable panel that you can use in any dashboard.
+// When you make a change to a library panel, that change propagates to all instances of where the panel is used.
+// Library panels streamline reuse of panels across multiple dashboards.
+LibraryPanelRef: {
   // Library panel name
   name: string
```

Comment:
Think we agreed this isn't needed?

Reply:
Would you remove just the name? or the nesting too?

We could have something like: 
```json
{
  title: 'my title',
  id: 5
  libraryPanelUid: 'fooo'
}
```

Reply:
Remove just the name

Reply:
@dprokop While removing the `name` I faced multiple places where the name is used. So I'll move those changes into a different PR